ServerCore/ServerMessages/ServerMessageTypes.cs

```diff
@@ -107,4 +107,9 @@ public class Notification
     /// The link that should be followed if the notification is clicked.
     /// </summary>
     public string LinkUrl { get; set; }
+
+    /// <summary>
+    /// Says whether the notification is so critical that it should not disappear automatically
+    /// </summary>
+    public bool IsCritical { get; set; }
```

Comment:
Do we want to additionally add a parameter for players to suppress non-critical notifications by type?

Reply:
I don't understand what you mean by this, can you please elaborate?

Reply:
I mean if we want to give every notification a "category" (e.g. PuzzleThread, GameControlMessage, PuzzleSolves). Then players can be given an option of which ones they want to leave on and which ones they want to leave off

Reply:
Makes sense, thanks! I will make a separate issue for this rather than do it now, as I don't feel like doing the client UX at present.